orchestra/todos/views.py

```diff
@@ -37,13 +40,34 @@ def update_todos_from_todolist_template(request):
         raise BadRequest('TodoList Template not found for the given slug.')
 
 
+@api_endpoint(methods=['GET'],
+              permissions=(permissions.IsAuthenticated,
+                           IsAssociatedWithProject),
+              logger=logger)
+def recommendations(request):
+    try:
+        todoQAs = TodoQA.objects.filter(
+            todo__task__assignments__worker__user=request.user, approved=False)
+        recommendation_list = {todoQA.todo.description: TodoQASerializer(
+            todoQA).data for todoQA in todoQAs}
+    except TodoQA.DoesNotExist:
+        recommendation_list = {}
+    return Response(recommendation_list)
+
+
 class TodoList(generics.ListCreateAPIView):
     permission_classes = (permissions.IsAuthenticated,
                           IsAssociatedWithProject)
 
-    serializer_class = TodoSerializer
     queryset = Todo.objects.all()
 
+    def get_serializer_class(self):
```

Comment:
why this change to Todo?

Reply:
Only include todo QA data for users in the `project_admins` group.